pkgs/by-name/tl/tlsrouter/package.nix

```
@@ -0,0 +1,33 @@
+{
+  buildGo124Module,
+  fetchFromGitHub,
+  lib,
+  nix-update-script,
+}:
+
+buildGo124Module {
```

```suggestion
buildGoModule {
```
Unless you want to manually keep changing to newer version of go each time / unless on the lastest version of go it doesn't work, this should be `buildGoModule`  to default to the latest available.

---

changed the commit message and put your change in. thx